### ISSUE TITLE
Suggestion: place pointer inside of function call

### ISSUE THREAD

**Issue by Symmettry** [face-hh/griddycode#25]
When autocompleting a function it will put you at the end of the parenthesis

like doing

waf -> waffle() will put your cursor at the end so you type "waffle()1" instead of in the parenthesis of "waffle(1)", which is annoying

**Comment by face-hh** [comment]
Should be fixable by moving the caret a character backwards on function auto complete

**Comment by Symmettry** [comment]
aight lemme test

**Comment by Symmettry** [comment]
wait where do i do that

**Comment by face-hh** [comment]
settings.gd I believe, around the end in the "autocomplete" function

**Comment by face-hh** [comment]
There's no signal in Godot attached to CodeEdit that's called on autocomplete accept, so without a hacky method I doubt this is possible

**Comment by Symmettry** [comment]
yeah the only way i could think is one that’s very annoying to implement
